index.js

```diff
@@ -1,60 +1,29 @@
-const {app, BrowserWindow, ipcMain, Menu, shell} = require('electron')
 const defaultMenu = require('electron-default-menu')
+const window = require('electron-window')
+const electron = require('electron')
+const Env = require('envobj')
+const path = require('path')
 
-const env = process.env.NODE_ENV
+const delegateEvents = require('./lib/delegate-electron-events')
 
-let win, file, link
+const shell = electron.shell
+const Menu = electron.Menu
+const app = electron.app
```

Comment:
Maybe we want to come up with a style guide that we all agree on, I'd reckon we stick with standard style, and I'm not sure what the take is on this. As long as I don't have to use babel to compile anything, I'm fine using some of this ES6 stuff.

Wow that code in that tweet is horrific..